config/standard/app_settings.json

```diff
@@ -241,6 +241,14 @@
       "message": ""
     }
   ],
+  "replications": [
+    {
+      "from": "medic-user-[^\\-]+-meta",
+      "to": "medic-users-meta",
```

Comment:
I just realised if we don't hardcode the to db name then the default configuration becomes hard. Maybe this needs to be a db name suffix and we can prefix it with the `environment.db` at runtime?

Reply:
done